ceph-pull-requests/config/definitions/ceph-pull-requests.yml

```diff
@@ -46,7 +46,7 @@
           wipe-workspace: true
 
     builders:
-      - shell: "timeout 7200 ./run-make-check.sh"
+      - shell: "export NPROC=$(nproc); timeout 7200 ./run-make-check.sh"
```

Comment:
That's from coreutils, so that's safe and this is what run-make-check is also using. So that's not a new dep.